lib/Event/Model/ElementEventInterface.php

```diff
@@ -22,4 +22,15 @@ interface ElementEventInterface
      * @return ElementInterface
      */
     public function getElement();
+
+    /**
+     * Get argument by key.
+     *
+     * @param string $key Key
+     *
+     * @return mixed Contents of array key
+     *
+     * @throws \InvalidArgumentException If key is not found.
+     */
+    public function getArgument($key);
```

Comment:
I'd leave that out for this PR (BC-break).

Reply:
https://github.com/pimcore/pimcore/pull/7547